src/world/area_flo/flo_10/flo_10_4_fountain.c

```diff
@@ -259,47 +259,53 @@ EvtScript N(EVS_SetupFountain) = {
         Call(ModifyColliderFlags, MODIFY_COLLIDER_FLAGS_SET_BITS, COLLIDER_o86, COLLIDER_FLAGS_UPPER_MASK)
         Call(ModifyColliderFlags, MODIFY_COLLIDER_FLAGS_SET_BITS, COLLIDER_o87, COLLIDER_FLAGS_UPPER_MASK)
     EndIf
-    Call(EnableTexPanning, MODEL_o25, TRUE)
-    Call(EnableTexPanning, MODEL_o40, TRUE)
-    Call(EnableTexPanning, MODEL_o78, TRUE)
-    Call(EnableTexPanning, MODEL_o77, TRUE)
-    Call(EnableTexPanning, MODEL_o118, TRUE)
-    Call(EnableTexPanning, MODEL_o119, TRUE)
+    // fountain
+    Call(SetTexPanner, MODEL_o25, TEX_PANNER_1)
     Thread
         TEX_PAN_PARAMS_ID(TEX_PANNER_1)
         TEX_PAN_PARAMS_STEP(    0,-2000,    0,    0)
         TEX_PAN_PARAMS_FREQ(    0,    1,    0,    0)
         TEX_PAN_PARAMS_INIT(    0,    0,    0,    0)
         Exec(N(EVS_UpdateTexturePan))
     EndThread
+    // water edge
+    Call(SetTexPanner, MODEL_o40, TEX_PANNER_3)
     Thread
         TEX_PAN_PARAMS_ID(TEX_PANNER_3)
         TEX_PAN_PARAMS_STEP( -100,   80,    0,    0)
         TEX_PAN_PARAMS_FREQ(    1,    1,    0,    0)
         TEX_PAN_PARAMS_INIT(    0,    0,    0,    0)
         Exec(N(EVS_UpdateTexturePan))
     EndThread
+    // fountain mist
+    Call(SetTexPanner, MODEL_o78, TEX_PANNER_5)
     Thread
         TEX_PAN_PARAMS_ID(TEX_PANNER_5)
         TEX_PAN_PARAMS_STEP(-1000, 2000,    0,    0)
         TEX_PAN_PARAMS_FREQ(    1,    1,    0,    0)
         TEX_PAN_PARAMS_INIT(    0,    0,    0,    0)
         Exec(N(EVS_UpdateTexturePan))
     EndThread
+    // surface spread
+    Call(SetTexPanner, MODEL_o77, TEX_PANNER_6)
     Thread
         TEX_PAN_PARAMS_ID(TEX_PANNER_6)
         TEX_PAN_PARAMS_STEP(    0,-2000,    0,    0)
         TEX_PAN_PARAMS_FREQ(    0,    1,    0,    0)
         TEX_PAN_PARAMS_INIT(    0,    0,    0,    0)
         Exec(N(EVS_UpdateTexturePan))
     EndThread
+    // small fountain
+    Call(SetTexPanner, MODEL_o118, TEX_PANNER_8)
     Thread
         TEX_PAN_PARAMS_ID(TEX_PANNER_8)
         TEX_PAN_PARAMS_STEP(    0,-1500,    0,    0)
         TEX_PAN_PARAMS_FREQ(    0,    1,    0,    0)
         TEX_PAN_PARAMS_INIT(    0,    0,    0,    0)
         Exec(N(EVS_UpdateTexturePan))
     EndThread
+    // surface spread
+    Call(SetTexPanner, MODEL_o119, TEX_PANNER_9)
```

Comment:
would be nice to name the models....

Reply:
Perhaps, though they are one of the few things we have actual dev names for.